funtofem/optimization/optimization_manager.py

```diff
@@ -192,11 +192,12 @@ def _gatekeeper(self, x_dict):
                     self.comm, func_file, full_precision=False, optim=True
                 )
                 # copy the hotstart file to the checkpoints folder
-                src = self.hot_start_file
-                dest = os.path.join(
-                    self._checkpoints_folder, f"hot_start{self._iteration}.hst"
-                )
-                shutil.copy(src, dest)
+                if self.hot_start_file:
```

Comment:
```suggestion
                if self.hot_start_file is not None:
```

Reply:
Would it be better to form it like this?

Reply:
It's equivalent but more readable in the second way.

Reply:
I went ahead and made your change using your comment.